.github/workflows/ci.yml

```diff
@@ -27,7 +27,7 @@ jobs:
 
     strategy:
       matrix:
-        go-version: [1.17]
+        go-version: [1.18]
```

Comment:
Why the version bump? Since we are using version 1.17 as the minimum version in the go module, we should still test against it. You could just add version 1.18 to the list to test against both, or we could just bump the minimum go version in the module.

Reply:
One of the imports was at 1.18, but the go version in the test environment was 1.17, so the tests were failing. I didn't want to upgrade the repo, because I didn't know if there was a particular way that you wanted to handle that. How should I proceed?

Reply:
If the client tests are failing for go 1.17, we need to bump the minimum version in the module to 1.18. You can just run `go mod edit -go=1.18`.